CashSDK/src/test/java/cash/just/sdk/integration/AtmIntegrationCashCodeTest.kt

```diff
@@ -119,7 +116,7 @@ class AtmIntegrationCashCodeTest : AbstractAtmIntegrationTest() {
         Assert.assertFalse("isSuccessful$error", cashCodeStatusResponse.isSuccessful)
         Assert.assertTrue("Error Empty $error", StringUtils.isBlank(error))
         println("cashCodeResponse.body() : $cashCodeStatusResponse")
-        Assert.assertEquals("Result OK", "500", cashCodeStatusResponse.message())
```

Comment:
@anasoid It looks like the server has changed this logic from a 500 to 404, does it look right to you?

Reply:
No, 404 if endpoint are changing url

Reply:
what do you mean by that?